### DISCUSSION TITLE
[Feature Request] Block Services for Specific Devices  (not just all devices)

### DISCUSSION THREAD

**Discussion by Ltek** [AdguardTeam/AdGuardHome#7503] [Ideas]
Would like to have Block Services option that is Device Specific.  So we could have a Block Services page for each separate device (as needed). This will greatly help with making sure kids and/or guests do not get to some stuff while the admins, adults, etc can get to things they need.

thank you for the fantastic software!

**Comment by ainar-g** [comment]
You can already do that with persistent clients.

**Reply by Ltek** [reply]
> You can already do that with persistent clients.

Thank you! I missed that. It is pretty deep (hidden kind of)... Would be helpful if there was a direct link on the main Block Services page saying something like... "Want to set client specific Block Services? Click here."

I suspect 'persistent' is not an obvious term/meaning. All the settings anywhere in the app are actually 'persistent' (by definition of the word) since once set, they do not change  ;-)
